app/blacklist.json

```diff
@@ -0,0 +1,10 @@
+{
+  "generator-express-angular": "haven't updated package.json",
+  "ft-wp": "haven't updated package.json",
+  "generator-angular-phonegap": "haven't updated package.json",
+  "generator-angular-js": "haven't updated package.json",
+  "generator-ionicjs": "duplicate entry of generator-ionic",
+  "generator-charcoal": "deprecated",
+  "generator-react-coffee-webpack": "duplicate entry of generator-react-webpack",
+  "generator-angular-rails-proxy": "didn't change their package.json (ಠ_ಠ)"
+}
```

Comment:
An array is simpler and faster for us. They can easily look at the commit log if they're curious, though I bet they would never even care as most blacklisted stuff will be blacklisted because it's wrong and *abandoned*.